heron/metricscachemgr/src/java/com/twitter/heron/metricscachemgr/metricscache/MetricsCacheQueryUtils.java

```diff
@@ -0,0 +1,290 @@
+// Copyright 2016 Twitter. All rights reserved.
```

Comment:
To be clear my suggestion is whether or not this review needs to create classes like MetricsDatapoint and ExceptionDatapoint, or whether this implementation can just use the protobuf objects.